src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_adexchangebuyer.v1.4.json.baseline

```diff
@@ -61,31 +56,27 @@ puts JSON.pretty_generate(response)
 #    and check the quota for your project at
 #    https://console.developers.google.com/apis/api/adexchangebuyer
 # 2. Install the Ruby client library by running `gem install google-api-client`
+
 require 'googleauth'
 require 'google/apis/adexchangebuyer_v1_4'
 require 'json'
 
 service = Google::Apis::AdexchangebuyerV1_4::AdExchangeBuyerService.new
 
-# TODO: Change placeholder below to get authentication credentials.
-# See: https://foo.com/bar
+# TODO: Change placeholder below to generate authentication credentials. See:
+# https://foo.com/bar
 #
-# To use the `patch_account` method, authorize using one of the following scopes:
+# Authorize using the following scope:
 #   https://www.googleapis.com/auth/adexchange.buyer
 service.authorization = nil
 
-
-# TODO: Change placeholder below to desired parameter value for the `patch_account` method:
-
 # The account id
-id = 0
-
+id = 0  # TODO: Update placeholder value.
 
+# TODO: Assign values to desired members of `request_body`. Only assigned
+# fields will be changed:
```

Comment:
It's a style decision we went for before, so I'll keep it as is to be in line with the other languages. If you feel strongly I can make an issue to reconsider